build/build-validation.js

```diff
@@ -18,6 +18,7 @@ const defaultInitOptions = {
   bodyLimit: 1024 * 1024, // 1 MiB
   caseSensitive: true,
   disableRequestLogging: false,
+  jsonShorthand: true,
```

Comment:
I don't see a description of this option in the documentation. As such, I honestly have no idea what this PR accomplishes. Please update the docs.

Reply:
Where would I put additional documentation on it? Outside the context of JTD it just changes the internal behavior.
I added passing mention of it as it relates to JTD since it does allow for implementing it yourself, but it doesn't really make JTD "supported" yet, so I'd avoid saying too much about it, unless there's somewhere else I should add an explanation for people implementing their own custom shema validator constrors.

Reply:
https://github.com/fastify/fastify/blob/9256e72ff843cf002c9ebd2d2409631917975c0a/docs/Server.md#factory

Reply:
Added it here https://github.com/fastify/fastify/blob/8b159c9c0aff5675ac5668bcb38cb06febabef61/docs/Server.md#schema-json-shorthand